### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/advanced/presets.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#639] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/advanced/presets.html

为了满足不同用户的需求，主题提供了一些预设，你可以在 vuepress-theme-hope/presets 下获取它们并自行导入。 组件相关 为了通过别名替换组件，你需要将 { custom: true } 作为第二个选项传入 hopeTheme。 必应壁纸 将博客主页的背景替换为每日的必应壁纸。 组件: 

https://theme-hope.vuejs.press/zh/guide/advanced/presets.html

**Comment by cuckoocry** [comment]
照着代码写，不对呢？

**Reply by cuckoocry** [reply]
弄出來了，撤不回評論了

**Reply by lmy668** [reply]
你试了这个吗，主页背景替换，我的依赖路径 咋没有这个文件呢？
import BlogHero from "vuepress-theme-hope/blog/components/BlogHero.js"
